src/gui/FxMixerView.cpp

```diff
@@ -185,6 +185,8 @@ void FxMixerView::addNewChannel()
 	updateFxLine(newChannelIndex);
```

Comment:
I can include a commit to rename the Line terminology to Channel, but I need to understand if this is necessary/viable.